packages/odyssey-react/src/components/Select/Select.module.scss

```diff
@@ -12,12 +12,69 @@
 
 /* stylelint-disable color-no-hex, declaration-no-important, selector-class-pattern, max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, no-descending-specificity, selector-max-class, selector-max-type */
 
-/* - styles from _select.scss - */
-
 .root:not([data-type]) {
-  @include input-baseline;
+  @include border-radius(var(--BorderRadius));
 
   appearance: none;
+  display: block;
+  position: relative;
+  width: 100%;
+  max-width: var(--FontMaxWidth);
+  margin-block: var(--SpaceMarginBlock);
+  margin-inline: var(--SpaceMarginInline);
+  padding-block: var(--SpacePaddingBlock);
+  padding-inline: var(--SpacePaddingInline);
+  transition-property: border-color, background-color, box-shadow;
+  transition-duration: var(--TransitionDuration);
+  transition-timing-function: var(--TransitionTiming);
+  border-width: var(--BorderWidth);
+  border-style: var(--BorderStyle);
+  border-color: var(--ColorBorder);
+  background-color: var(--ColorBackground);
+  box-shadow: 0 0 0 0 cv("blue", "000");
```

Comment:
Do we have a palette token we can use here?

Reply:
I'll re-use the existing box-shadow theme variable here since it isn't visible anyways